gno.land/cmd/genesis/txs.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+type txsCfg struct {
+	genesisPath string
+}
+
+// newTxsCmd creates the genesis txs subcommand
+func newTxsCmd(io *commands.IO) *commands.Command {
+	cfg := &txsCfg{}
+
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "txs",
+			ShortUsage: "txs <subcommand> [flags]",
+			ShortHelp:  "Manages the initial genesis transactions",
+			LongHelp:   "Manages genesis transactions through input files",
+		},
+		cfg,
+		commands.HelpExec,
+	)
+
+	cmd.AddSubCommands(
+		newTxsAddCmd(cfg, io),
+		newTxsRemoveCmd(cfg, io),
+	)
+
+	return cmd
+}
+
+func (c *txsCfg) RegisterFlags(fs *flag.FlagSet) {
+	fs.StringVar(
```

Comment:
I suggest either moving the `genesis-path` flag to the top-level root command or creating a helper function to ensure it is consistently added throughout.

Reply:
I haven't noticed almost all of them share `genesis-path` 🤦‍♂️ 

Only `genesis generate` doesn't, so I've made this flag registration a helper:

[3f2d923](https://github.com/gnolang/gno/pull/1252/commits/3f2d923875558e6ce8c98369428b23f471410f38)